### PR TITLE
Conversions: Fix casing

### PR DESCRIPTION
**Pull request by GuiltyDolphin**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [X] Improvement
    - [X] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Fixes an issue with casing not being handled properly.

###### People to notify (@mention interested parties)

@mintsoft

---

Instant Answer Page: https://duck.co/ia/view/conversions

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mintsoft

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by GuiltyDolphin** [comment]
@mintsoft This is against your current plurals PR, so if you merge you'll probably want to re-base.

File: lib/DDG/Goodie/Conversions.pm [hunk 1]
```diff
@@ -19,11 +19,17 @@ my @types = LoadFile(share('ratios.yml'));
...
 my @units = ();
+my %unit_aliases;
 foreach my $type (@types) {
-    push(@units, $type->{'unit'});
-    push(@units, $type->{'plural'}) unless lc $type->{'unit'} eq lc $type->{'plural'};
-    push(@units, @{$type->{'aliases'}});
-    $plurals{lc $type->{'unit'}} = lc $type->{'plural'};
+    my $actual_unit = $type->{unit};
+    my $plural = $type->{plural};
+    my %unit_names = map { $_ => 1 } (
```
Comment (mintsoft) [inline lib/DDG/Goodie/Conversions.pm]: wtf is going on here!? :laughing:

Comment (GuiltyDolphin) [inline lib/DDG/Goodie/Conversions.pm]: Hmm?

Comment (mintsoft) [inline lib/DDG/Goodie/Conversions.pm]: It's the `map` that's confusing me!

Comment (GuiltyDolphin) [inline lib/DDG/Goodie/Conversions.pm]: @mintsoft Yeah, the map is just to build an even-sized list we can treat as a HASH.

**Comment by GuiltyDolphin** [comment]
Fixed in #3019.

**Pull request closed without merging**